content/docs/atom-guide/look-dev/vegetation-bending-tutorial.md

```diff
@@ -645,6 +645,10 @@ However, we don't have an vertex shader input that gives us the previous positio
 
 Great, now we have included the motion vector shader, so we have to go in and edit the vertex shader. Before that, however, we need to have a way to get the previous frame's time to use in our bending calculations. The `SceneSrg` has the current frame's time (`m_time`), but it doesn't contain the previous frame's time. Let's add the previous time in the `SceneSrg`.
 
+{{< note >}}
+We will be making some minor changes to the core engine here. This is generally not a best practice because there may be version updates, but we do this as a proof of concept for now. Additionally, the way that we will define the "previous time" also only works for the common case of a single pipeline with a single scene, but again this works for our proof of concept in this tutorial.
```

Comment:
I think it's worth explaining the reasoning for why it doesn't work for all cases here. Something along the lines of

"If you have two render pipelines that are part of the same scene, and both update in the same simulation frame, but one was last updated the previous simulation frame, and the other was last updated 3 simulation frames ago, they would have two different previous times, but would both be referring to the same SceneSrg."